### PR TITLE
Add -q flag for image ls command

### PR DESCRIPTION
**Pull request by jcsirot**
**- What I did**
This PR add the flag `-q` to the `docker app image ls` command. When this flag is set to true only the app image id are printed.

**- How to verify it**
Run `docker app image ls -q`. The output must look like:
```
$ docker app image ls -q
bfd17c107aad
1fa73eccfc5c
bfd17c107aad
bfd17c107aad
916c80e7ab93
```

A E2E test covering this command has been added

**- A picture of a cute animal (not mandatory but encouraged)**
image

### CODE REVIEW

File: internal/store/digest.go [hunk 1]
```diff
@@ -27,6 +28,11 @@ func FromString(s string) (ID, error) {
 	return ID{s}, nil
 }
 
+func FromBundle(bndle *bundle.Bundle) (ID, error) {
```
[the remark this answers is not in the document]
Comment (ndeloof) [reply internal/store/digest.go]: Indeed. What about attaching a getID() function to Bundle ?
(sorry, my brain is still object-oriented)

File: e2e/images_test.go [hunk 2]
```diff
@@ -44,6 +63,16 @@ b-simple-app:latest           simple
 	})
 }
 
+func TestImageListQuiet(t *testing.T) {
```
Comment (silvin-lubecki) [inline e2e/images_test.go]: I think a unit test would be easier for this feature, mocking the bundle store to configure it the way you want.

File: internal/store/digest.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (eunomie) [reply internal/store/digest.go]: wasn't the remark about `bndle`
Everywhere in the code it's `bndl`, `bndle` is used only inside `internal/store/bundle.go`

[the remark this answers is not in the document]
Comment (rumpl) [reply internal/store/digest.go]: Nah, `FromBundle` is an odd name. When called it's: `store.FromBundle(..)` and you get ... an ID. Confusing to say the least

[the remark this answers is not in the document]
Comment (eunomie) [reply internal/store/digest.go]: Maybe the package is not the right one.
Something like `id.FromString` and `id.FromBundle` (or digest)
Not `store.FromBundle` as we don't return a store.

File: internal/commands/image/list_test.go [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+	bundleStore := &bundleStoreStubForListCmd{
+		refMap:  make(map[reference.Reference]*bundle.Bundle),
+		refList: []reference.Reference{},
+	}
+	ref1, err := store.FromString("a855ac937f2ed375ba4396bbc49c4093e124da933acd2713fb9bc17d7562a087")
+	assert.NilError(t, err)
+	ref2, err := reference.Parse("foo/bar:1.0")
+	assert.NilError(t, err)
+	//nolint: errcheck
+	bundleStore.Store(ref1, &bundle.Bundle{})
```
Comment (silvin-lubecki) [inline internal/commands/image/list_test.go]: You should assert on the error here.

**Pull request merged**